packages/contracts/contracts/libs/LibStaticCall.sol

```diff
@@ -24,7 +25,7 @@ library StaticCall {
   {
     require(to.isContract(), "StaticCall to address is not a contract.");
     assembly {
-      let success := staticcall(gas, to, add(data, 0x20), mload(data), 0, 0)
+      success := staticcall(gas, to, add(data, 0x20), mload(data), 0, 0)
```

Comment:
Out of curiosity, what's the difference between these assembly stmts?

Reply:
I’m not sure how the first one passed since the success variable name is already in scope